qcore/validate_vm.py

```diff
@@ -51,12 +52,12 @@ def vmfile(filename):
     for fixed_name in vm.values():
         if not os.path.exists(fixed_name):
             return False, 'VM file not found: %s' % (fixed_name)
-    if not os.path.exists(vmfile('vm_params.yaml')):
+    if not os.path.exists(vmfile(VM_PARAMS_FILE_NAME)):
```

Comment:
While we are here it probably would be a good idea to move more of this logic to structure.py 
i.e. the VM file function too.

Reply:
Can also move the writing to this same location too.